os/board/rtl8730e/src/component/wifi/inic/inic_ipc_host_trx.c

```diff
@@ -309,6 +312,9 @@ void inic_ipc_host_rx_handler(int idx_wlan, struct sk_buff *skb)
 
 	/* allocate host_recv_buf and associate to the p_buf */
 	precvbuf = (struct host_recv_buf *)rtw_zmalloc(sizeof(struct host_recv_buf));
+	if (!precvbuf) {
+		goto RSP;
```

Comment:
I can't find pbuf free in this case. Could you check it?

Reply:
Hi @sunghan-chang ,

Added the call to free the pbuf 
/* free p_buf for the case: SUCCESS allocate pbuf but FAILED allocate precvbuf */

https://github.com/Samsung/TizenRT/compare/1207b1c2d6345ee7a8c7b1caaead340f116c265f..463ddef16a14b779afd017d80cfe2474b2913e14

Thank You.